Vagrantfile

```diff
@@ -0,0 +1,15 @@
+Vagrant.configure("2") do |config|
+    config.vm.box = "ubuntu/trusty64"
```

Comment:
This is trusty, i.e. Ubuntu 14.04. We want [xenial, i.e. Ubuntu 16.04](https://app.vagrantup.com/ubuntu/boxes/xenial64). See [ubuntu version history](https://en.wikipedia.org/wiki/Ubuntu_version_history) as a reference.